# Triggering an error inside a custom loader when querying the key of a field

-- input
field: value
-- input

-- javascript
eno.parse(input).field('field').key(() => { throw 'my error'; });
-- javascript

-- php
Eno\Parser::parse($input)->field('field')->key(function() { throw new Exception('my error'); });
-- php

-- python
def loader(value):
  raise ValueError('my error')

enolib.parse(input).field('field').key(loader)
-- python

-- ruby
Eno.parse(input).field('field').key { raise 'my error' }
-- ruby

## ValidationError

-- text
There is a problem with the key of this element: my error
-- text

-- snippet
   Line | Content
 >    1 | field: value
-- snippet

selection: [0,0] => [0,5]
